CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### Features
 
 * [593](https://github.com/provenance-io/cosmos-sdk/pull/593) Provenance: Use a new IMsgServiceRouter interface instead of a concrete one.
```

Comment:
Should we fix the changelog above?  To have the '#' as well?

Reply:
❤️ Fixed it in a different PR. Once I fix merge conflicts, it'll be GTG.